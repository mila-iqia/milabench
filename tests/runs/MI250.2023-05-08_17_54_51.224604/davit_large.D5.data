{"event": "config", "data": {"dirs": {"base": "/milabench/envs", "venv": "/milabench/envs/venv/torch", "data": "/milabench/envs/data", "runs": "/milabench/envs/runs", "extra": "/milabench/envs/extra/timm", "cache": "/milabench/envs/cache"}, "arch": "rocm", "group": "timm", "install_group": "torch", "install_variant": "rocm", "run_name": "redizilo.2023-05-08_17:54:51.224604", "enabled": true, "capabilities": {"nodes": 1}, "max_duration": 600, "voir": {"options": {"stop": 60, "interval": "1s"}}, "config_base": "/milabench/milabench/config", "config_file": "/milabench/milabench/config/standard.yaml", "definition": "/milabench/milabench/benchmarks/timm", "plan": {"method": "per_gpu"}, "argv": {"--amp": true, "--model": "davit_large", "--batch-size": 128, "--lr-base": 0.01}, "tags": ["classification", "transformer", "vision"], "weight": 1.0, "name": "davit_large", "tag": ["davit_large", "D5"], "device": 5, "devices": [5], "env": {"ROCR_VISIBLE_DEVICES": "5"}}, "pipe": null}
{"event": "start", "data": {"command": ["voir", "--config", "/milabench/envs/extra/timm/voirconf-davit_large.D5-0efae956f1553a76c1e03985181900f5.json", "/milabench/milabench/benchmarks/timm/pytorch-image-models/train.py", "--amp", "--model", "davit_large", "--batch-size", "128", "--lr-base", "0.01", "--data-dir", "/milabench/envs/data", "--dataset", "FakeImageNet", "--output", "/milabench/envs/extra/timm/redizilo.2023-05-08_17:54:51.224604/davit_large.D5", "--checkpoint-hist", "1"], "time": 1683570775.336998}, "pipe": null}
{"event": "phase", "data": {"name": "init"}, "pipe": "data"}
{"event": "phase", "data": {"name": "parse_args"}, "pipe": "data"}
{"event": "phase", "data": {"name": "load_script"}, "pipe": "data"}
{"event": "phase", "data": {"name": "run_script"}, "pipe": "data"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "Training with a single process on 1 device (cuda:0).\n", "pipe": "stderr"}
{"event": "line", "data": "Model davit_large created, param count:196811752\n", "pipe": "stderr"}
{"event": "line", "data": "Data processing configuration for current model + dataset:\n", "pipe": "stderr"}
{"event": "line", "data": "\tinput_size: (3, 224, 224)\n", "pipe": "stderr"}
{"event": "line", "data": "\tinterpolation: bicubic\n", "pipe": "stderr"}
{"event": "line", "data": "\tmean: (0.485, 0.456, 0.406)\n", "pipe": "stderr"}
{"event": "line", "data": "\tstd: (0.229, 0.224, 0.225)\n", "pipe": "stderr"}
{"event": "line", "data": "\tcrop_pct: 0.95\n", "pipe": "stderr"}
{"event": "line", "data": "\tcrop_mode: center\n", "pipe": "stderr"}
{"event": "line", "data": "Learning rate (0.005) calculated from base learning rate (0.01) and global batch size (128) with linear scaling.\n", "pipe": "stderr"}
{"event": "line", "data": "Using native Torch AMP. Training in mixed precision.\n", "pipe": "stderr"}
{"event": "line", "data": "Scheduled epochs: 300. LR stepped per epoch.\n", "pipe": "stderr"}
{"event": "data", "data": {"task": "main", "gpudata": {"5": {"memory": [839, 65520], "load": 0.0, "temperature": 33.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.224273681640625}, "pipe": "data"}
{"event": "line", "data": "Train: 0 [   0/32 (  0%)]  Loss: 7.224 (7.22)  Time: 5.701s,   22.45/s  (5.701s,   22.45/s)  LR: 1.000e-05  Data: 0.738 (0.738)\n", "pipe": "stderr"}
{"event": "data", "data": {"task": "main", "gpudata": {"5": {"memory": [22618, 65520], "load": 0.39, "temperature": 37.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.17632532119751}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"5": {"memory": [32306, 65520], "load": 0.47, "temperature": 39.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.2762956619262695}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.335553169250488}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"5": {"memory": [32306, 65520], "load": 1.0, "temperature": 40.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.256007671356201}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"5": {"memory": [32306, 65520], "load": 1.0, "temperature": 42.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.162002086639404}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"5": {"memory": [32306, 65520], "load": 1.0, "temperature": 42.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.256006240844727}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.51413265075542, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [1, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"5": {"memory": [32306, 65520], "load": 0.42, "temperature": 42.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.163412094116211}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.50927821582911, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [2, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.308283805847168}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"5": {"memory": [32308, 65520], "load": 1.0, "temperature": 42.0}}}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.29058605429223, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [3, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.2834391593933105}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"5": {"memory": [32308, 65520], "load": 1.0, "temperature": 43.0}}}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.61200437826779, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [4, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.234735012054443}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"5": {"memory": [32308, 65520], "load": 1.0, "temperature": 43.0}}}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.68063191177269, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [5, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.196158409118652}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.487314703434535, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [6, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"5": {"memory": [32308, 65520], "load": 0.3, "temperature": 43.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.325285911560059}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.55540400277077, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [7, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.243382453918457}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"5": {"memory": [32308, 65520], "load": 1.0, "temperature": 43.0}}}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.394100614247265, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [8, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.183416366577148}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"5": {"memory": [32308, 65520], "load": 1.0, "temperature": 43.0}}}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.44719282177041, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [9, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.22973108291626}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"5": {"memory": [32308, 65520], "load": 1.0, "temperature": 43.0}}}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.71837618638708, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [10, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.048616409301758}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.78482782110971, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [11, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"5": {"memory": [32308, 65520], "load": 0.8, "temperature": 41.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.186925888061523}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.716159440061965, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [12, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.3048930168151855}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"5": {"memory": [32308, 65520], "load": 1.0, "temperature": 42.0}}}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.77243144748587, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [13, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.246750831604004}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"5": {"memory": [32308, 65520], "load": 1.0, "temperature": 43.0}}}, "pipe": "data"}
{"event": "line", "data": "Memory access fault by GPU node-7 (Agent handle: 0x853f040) on address 0x7fef5fc00000. Reason: Unknown.\n", "pipe": "stderr"}
{"event": "end", "data": {"command": ["voir", "--config", "/milabench/envs/extra/timm/voirconf-davit_large.D5-0efae956f1553a76c1e03985181900f5.json", "/milabench/milabench/benchmarks/timm/pytorch-image-models/train.py", "--amp", "--model", "davit_large", "--batch-size", "128", "--lr-base", "0.01", "--data-dir", "/milabench/envs/data", "--dataset", "FakeImageNet", "--output", "/milabench/envs/extra/timm/redizilo.2023-05-08_17:54:51.224604/davit_large.D5", "--checkpoint-hist", "1"], "time": 1683570844.7449741, "return_code": -6}, "pipe": null}
