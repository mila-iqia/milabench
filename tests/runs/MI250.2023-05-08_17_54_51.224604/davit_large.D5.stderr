/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
Training with a single process on 1 device (cuda:0).
Model davit_large created, param count:196811752
Data processing configuration for current model + dataset:
	input_size: (3, 224, 224)
	interpolation: bicubic
	mean: (0.485, 0.456, 0.406)
	std: (0.229, 0.224, 0.225)
	crop_pct: 0.95
	crop_mode: center
Learning rate (0.005) calculated from base learning rate (0.01) and global batch size (128) with linear scaling.
Using native Torch AMP. Training in mixed precision.
Scheduled epochs: 300. LR stepped per epoch.
Train: 0 [   0/32 (  0%)]  Loss: 7.224 (7.22)  Time: 5.701s,   22.45/s  (5.701s,   22.45/s)  LR: 1.000e-05  Data: 0.738 (0.738)
Memory access fault by GPU node-7 (Agent handle: 0x853f040) on address 0x7fef5fc00000. Reason: Unknown.
